app/models/vmdb_database_connection.rb

```diff
@@ -25,25 +25,40 @@ class VmdbDatabaseConnection < ApplicationRecord
   virtual_column :blocked_by, :type => :integer
 
   def self.log_statistics(output = $log)
+    log_activity(output)
+    log_table_size(output)
+    log_table_statistics(output)
+  end
+
+  def self.log_csv(keys, stats, label, output)
```

Comment:
Probably should be 
```suggestion
  private_class_method def self.log_csv(keys, stats, label, output)
```

Reply:
I think it was done as a CSV to avoid having to print the same keys over and over again.  If you do a hash, at least half of each row of data will be printing the keys, unless we abbreviate them.

Reply:
@Fryguy I'm fine with changing formats.  The reason CSV was used was because of the single header row instead of repeating the same header values in other structures such as keys in a hash or json.  As it is though, the log line limit gets reached so we can't even log the CSV in a single line so we'll need to split up anyway.